src/newmatrix.cpp

```diff
@@ -834,10 +837,25 @@ void matrix_fight(struct matrix_icon *icon, struct matrix_icon *targ)
         bod = targ->decker->masking;
         break;
       }
+      if (targ->type == ICON_LIVING_PERSONA)
+        bod += targ->decker->hardening; // Otaku get to add their hardening to this check.
```

Comment:
They do? This feels OP, can't they get to hardening-12 for a completely unreachable bod number?